rust/ql/test/library-tests/dataflow/sources/InlineFlow.ql

```diff
@@ -0,0 +1,21 @@
+import rust
+import codeql.rust.dataflow.DataFlow
+import codeql.rust.Concepts
+import utils.InlineFlowTest
+
+/**
+ * Configuration for flow from any threat model source to an argument of a function called `sink`.
+ */
+module MyFlowConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node source) { source instanceof ThreatModelSource }
+
+  predicate isSink(DataFlow::Node sink) {
+    any(CallExpr call | call.getExpr().(PathExpr).getPath().toString() = "sink")
```

Comment:
Should this be `getResolvedPath()` instead of `toString()`. Relying on `toString()` seems a bit brittle.

Reply:
Fixed - thanks for providing the answer here.

There's at least one more place we're abusing `toString` at the moment (not in this PR), I created an issue about it a couple of days ago.  So don't worry, this is not a bad habit we're getting into!